### DISCUSSION TITLE
Mania notes stop scrolling and after a couple of second all notes scroll at once

### DISCUSSION THREAD

**Discussion by reo-the-enby** [ppy/osu#23435] [Q&A]
### Type

Game behaviour

### Bug description

The notes stopped scrolling while the song still going and after some seconds it caught up and the notes who hadn't scrolled down all came at once and it resulted in a fail.

I didn't record it but can "hopefully" be reproduced

### Version

2023.419.0

### Logs

[database.log](https://github.com/ppy/osu/files/11404540/database.log)
[input.log](https://github.com/ppy/osu/files/11404541/input.log)
[network.log](https://github.com/ppy/osu/files/11404542/network.log)
[performance-audio.log](https://github.com/ppy/osu/files/11404543/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11404544/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11404545/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11404546/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/11404547/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11404548/runtime.log)

**Comment by bdach** [comment]
> I didn't record it but can "hopefully" be reproduced

Unlikely it will, given that you didn't even link the beatmap, or _anything_. Basically zero information to go on here. Logs won't help _if I don't know which part of them I'm even supposed to be reading_.

**Comment by reo-the-enby** [comment]
I was playing this map https://osu.ppy.sh/beatmapsets/1565587#mania/3196930 with HD and Wind Up set to 1,00x to 1,38x
